src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -65,7 +65,7 @@ public PersistentDataStore() {
         UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
         String userName = (String) entity.getProperty("username");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        User user = new User(uuid, userName, creationTime);
+        User user = new User(uuid, userName, null, creationTime); //To DO: get Password from Datastore; Currently null
```

Comment:
We want to put comments on a separate line unless its a very short comment i.e. less than 4 words.

Reply:
Will fix it.